### ISSUE TITLE
no default export.ts

### ISSUE THREAD

**Issue by StErMi** [kerimdzhanov/dotenv-flow#38]
This is my `package.json`

```
{
    "dependencies": {
        "dotenv-flow": "^3.2.0"
    },
    "devDependencies": {
        "@types/dotenv-flow": "^3.1.0"
    }
}
```

This is the code:

```
import dotenvFlow from 'dotenv-flow';
dotenvFlow.config();
```

This is the full error message:

```
Module '"/Users/<PROJECT>/cdk/node_modules/@types/dotenv-flow/index"' has no default export.ts(1192)
```

how can I fix it?

**Comment by kerimdzhanov** [comment]
Hi @StErMi,

Thanks for the issue.

It seems that we have to fix the [typings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/dotenv-flow).
Let's left this issue open. I'm going to work on this later.

For now, you can fix it using the named import, like this:

```
import { config } from 'dotenv-flow';
config();
```

**Comment by kerimdzhanov** [comment]
Closed in #57.
